catalogs/catalog-lakehouse-paimon/src/main/java/org/apache/gravitino/catalog/lakehouse/paimon/utils/TableOpsUtils.java

```diff
@@ -18,20 +18,124 @@
+  public static List<SchemaChange> buildSchemaChanges(TableChange... tableChanges)
+      throws UnsupportedOperationException {
+    List<SchemaChange> schemaChanges = new ArrayList<>();
+    for (TableChange tableChange : tableChanges) {
+      schemaChanges.add(buildSchemaChange(tableChange));
+    }
+    return schemaChanges;
+  }
+
+  public static SchemaChange buildSchemaChange(TableChange tableChange)
+      throws UnsupportedOperationException {
+    if (tableChange instanceof ColumnChange) {
+      if (tableChange instanceof AddColumn) {
+        AddColumn addColumn = (AddColumn) tableChange;
+        String fieldName = getfieldName(addColumn);
+        checkColumnCapability(fieldName, addColumn.getDefaultValue(), addColumn.isAutoIncrement());
+        return addColumn(
+            fieldName,
+            toPaimonType(addColumn.getDataType()).copy(addColumn.isNullable()),
+            addColumn.getComment(),
+            move(fieldName, addColumn.getPosition()));
+      } else if (tableChange instanceof DeleteColumn) {
+        return dropColumn(getfieldName((DeleteColumn) tableChange));
+      } else if (tableChange instanceof RenameColumn) {
+        RenameColumn renameColumn = ((RenameColumn) tableChange);
+        return renameColumn(getfieldName(renameColumn), renameColumn.getNewName());
+      } else if (tableChange instanceof UpdateColumnComment) {
+        UpdateColumnComment updateColumnComment = (UpdateColumnComment) tableChange;
+        return updateColumnComment(
+            getfieldName(updateColumnComment), updateColumnComment.getNewComment());
+      } else if (tableChange instanceof UpdateColumnNullability) {
+        UpdateColumnNullability updateColumnNullability = (UpdateColumnNullability) tableChange;
+        return updateColumnNullability(
+            getfieldName(updateColumnNullability), updateColumnNullability.nullable());
+      } else if (tableChange instanceof UpdateColumnPosition) {
+        UpdateColumnPosition updateColumnPosition = (UpdateColumnPosition) tableChange;
+        Preconditions.checkArgument(
+            !(updateColumnPosition.getPosition() instanceof Default),
+            "Default position is not supported for Paimon update column position.");
+        return updateColumnPosition(
+            move(getfieldName(updateColumnPosition), updateColumnPosition.getPosition()));
+      } else if (tableChange instanceof UpdateColumnType) {
+        UpdateColumnType updateColumnType = (UpdateColumnType) tableChange;
+        return updateColumnType(
+            getfieldName(updateColumnType), toPaimonType(updateColumnType.getNewDataType()));
+      }
+    } else if (tableChange instanceof UpdateComment) {
+      return updateComment(((UpdateComment) tableChange).getNewComment());
+    } else if (tableChange instanceof SetProperty) {
+      SetProperty setProperty = ((SetProperty) tableChange);
+      String setPropertyKey = setProperty.getProperty();
+      if (COMMENT.equals(setPropertyKey)) {
```

Comment:
please remove the `comment` logic in `SetProperty` and `RemoveProperty`

Reply:
done